docs/cudf/source/cudf_pandas/how-it-works.md

```diff
@@ -36,3 +36,16 @@ transfers.
 When using `cudf.pandas`, cuDF's [pandas compatibility
 mode](api.options) is automatically enabled, ensuring consistency with
 pandas-specific semantics like default sort ordering.
+
+
+`cudf.pandas` uses a managed pool memory by default, that will also enable
+prefetching <<LINK to prefetch tutorial/demo>>.
+
+There are various memory allocators that can be used by changing the environment
+variable `CUDF_PANDAS_RMM_MODE`. It supports:
+
+1. "pool"
+2. "async"
+3. "managed" (default)
+4. "managed_pool"
+5. "cuda"
```

Comment:
I believe `managed_pool` is the default, not `managed`? Also suggesting more detail, and reordered (sort of) in order of preferred use (?).

```suggestion
Other memory allocators can be used by changing the environment
variable `CUDF_PANDAS_RMM_MODE` to one of the following.

Pool allocators improve allocation performance. Without using one, memory 
allocation may be a bottleneck depending on the workload. Managed memory
enables oversubscribing GPU memory. This allows cudf.pandas to process 
data larger than GPU memory in many cases, without CPU (Pandas) fallback.

1. "managed_pool" (default): CUDA Unified Memory (managed memory) with RMM's asynchronous pool allocator.
2. "managed": CUDA Unified Memory, (managed memory) with no pool allocator.
3. "async": CUDA's built-in pool asynchronous pool allocator with normal CUDA device memory. 
4. "pool": RMM's asynchronous pool allocator with normal CUDA device memory.
5. "cuda": normal CUDA device memory with no pool allocator.
```